packages/fx-core/tests/common/local/localSettingsHelper.test.ts

```diff
@@ -86,14 +76,22 @@ describe("localSettingsHelper", () => {
       await fs.ensureDir(projectPath);
       await fs.emptyDir(projectPath);
 
-      const projectSettingsAll = cloneDeep(projectSettings0);
-      const aadPluginKey = projectSettingsAll.solutionSettings.activeResourcePlugins.indexOf(
-        "fx-resource-aad-app-for-teams"
-      );
-      if (aadPluginKey > -1) {
-        projectSettingsAll.solutionSettings.activeResourcePlugins.splice(aadPluginKey, 1);
-      }
-      const localEnvs = await convertToLocalEnvs(projectPath, projectSettingsAll, localSettings0);
+      const projectSettings = {
```

Comment:
const projectSettings = cloneDeep(projectSettings0);
projectSettings.xxx = xxx...

Keep the clone logic for easier modification.

Reply:
It not simple to add some plugins or remove some plugins in v3 project settings. So defining a new projectSettings makes the sample easiler to understand. We can optimize the case in the futhure.